apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -1055,6 +1052,27 @@ maybe_run_fuzzy_filter(Rows, QString0) ->
             )
     end.
 
+%% These filters, while they are able to be adapted to efficient ETS match specs, must be
+%% used as fuzzy filters when iterating over offlient persistent clients, which live
+%% outside ETS.
+adapt_custom_filters(Qs, Fuzzy) ->
+    lists:foldl(
+        fun
+            ({Field, '=:=', X}, {QsAcc, FuzzyAcc}) when
```

Comment:
should mimic the `like` implementation?
binary:match/2

Reply:
No, this is an exact query, not a fuzzy one, because it can't be an ETS match clause when iterating over DS sessions.